src/sentry/conf/server.py

```diff
@@ -460,6 +461,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     Queue('events.reprocessing.process_event', routing_key='events.reprocessing.process_event'),
     Queue('events.reprocess_events', routing_key='events.reprocess_events'),
     Queue('events.save_event', routing_key='events.save_event'),
+    Queue('files', routing_key='files'),
```

Comment:
@JTCunning Should we maybe name this `files.delete` ? Or cross that bridge if/when it happens.

Reply:
Yes, we should.

Reply:
Ask and you shall recv! 60da051ed